src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -1706,6 +1706,49 @@ public void testViolationMessageOnIoException() throws Exception {
         verify(checkerConfig, filePath, expected);
     }
 
+    /**
+     * Test doesn't need to be serialized.
```

Comment:
`This test is created to cover pitest mutation over Error message. Usually Exceptions are thrown , not Error, so we need to mock creation of Error.`

Reply:
done